### PR TITLE
Added template for CVE-2024-1380

### PR DESCRIPTION
**Pull request by pdteamx**
Added template for CVE-2024-1380

### CODE REVIEW

**Comment by FLX-0x00** [comment]
I have added this through cloud.projectdiscovery.io so the template is not in the right directory. Sorry for that - first try on this feature directly from the cloud interface =)

File: CVE-2024-1380.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  reference:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-1380
+    - https://www.wordfence.com/threat-intel/vulnerabilities/id/7b2a3b17-0551-4e02-8e6a-ae8d46da0ef8?source=cve
+    - https://plugins.trac.wordpress.org/changeset?sfp_email=&sfph_mail=&reponame=&old=3033880%40relevanssi&new=3033880%40relevanssi&sfp_email=&sfph_mail=
+  classification:
+    cvss-metrics: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N
+    cvss-score: 5.3
+    cve-id: CVE-2024-1380
+    epss-score: 0.00043
+    epss-percentile: 0.0866
```
Comment (FLX-0x00) [inline CVE-2024-1380.yaml:18]: missing metadata

```yaml
  metadata:
    verified: true
    max-request: 1
    fofa-query: "/wp-content/plugins/relevanssi/"

```

File: CVE-2024-1380.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      Content-Type: application/x-www-form-urlencoded; charset=UTF-8
+
+    body: "action=&relevanssi_export=1"
+
+    matchers:
+      - type: dsl
+        dsl:
+          - 'status_code == 200'
+          - 'contains(header, "Content-Disposition: attachment;filename=relevanssi_log.csv")'
+          - 'contains(header, "Content-Type: application/download")'
```
Comment (FLX-0x00) [inline CVE-2024-1380.yaml:36]: add 
```yaml
condition: and
```

**Comment by ritikchaddha** [comment]
Hello @FLX-0x00, thank you so much for sharing this template with the community and contributing to this project 🍻

**Review by ritikchaddha — APPROVED** [review]

**Comment by FLX-0x00** [comment]
> Contributor

cool - Can you give me any information as to why you use the RAW method instead of the POST syntax? It is hard for me to get the right point when this is better or that is better.

**Review by DhiyaneshGeek — APPROVED** [review]

**Pull request merged**